daemon/analytics.c

```diff
@@ -0,0 +1,195 @@
+#include "common.h"
+
+static const char *verify_required_directory(const char *dir) {
+    if(chdir(dir) == -1)
+        fatal("Cannot cd to directory '%s'", dir);
```

Comment:
Please, instead to have the message "Cannot cd", could we have something like this "Cannot change to directory ..."? I think the current message is not clear for users.

Reply:
Ok, thanks! Fixed with https://github.com/netdata/netdata/pull/10887/commits/0f5189aa3341b6377d662165de84f3b15e64dd80